docs/pbc/all/miscellaneous/202407.0/tutorials-and-howtos/add-audit-log-types.md

```diff
@@ -0,0 +1,165 @@
+use Spryker\Shared\LogExtension\Dependency\Plugin\AuditLoggerConfigPluginInterface;
+use Spryker\Yves\Kernel\AbstractPlugin;
+use Spryker\Yves\Log\Plugin\Log\AuditLogMetaDataProcessorPlugin;
+use Spryker\Yves\Log\Plugin\Log\AuditLogRequestProcessorPlugin;
+use Spryker\Yves\Log\Plugin\Log\AuditLogTagFilterBufferedStreamHandlerPlugin;
+use Spryker\Yves\Log\Plugin\Processor\EnvironmentProcessorPlugin;
+use Spryker\Yves\Log\Plugin\Processor\PsrLogMessageProcessorPlugin;
+use Spryker\Yves\Log\Plugin\Processor\ResponseProcessorPlugin;
+use Spryker\Yves\Log\Plugin\Processor\ServerProcessorPlugin;
+
+class YvesCheckoutAuditLoggerConfigPlugin extends AbstractPlugin implements AuditLoggerConfigPluginInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\AuditLoggerConfigCriteriaTransfer $auditLoggerConfigCriteriaTransfer
+     *
+     * @return bool
+     */
+    public function isApplicable(AuditLoggerConfigCriteriaTransfer $auditLoggerConfigCriteriaTransfer): bool
+    {
+        return $auditLoggerConfigCriteriaTransfer->getChannelName() === $this->getChannelName();
+    }
+
+    /**
+     * @return string
+     */
+    public function getChannelName(): string
+    {
+        return 'checkout';
+    }
+
+    /**
+     * @return list<\Spryker\Shared\Log\Dependency\Plugin\LogHandlerPluginInterface>
+     */
+    public function getHandlers(): array
+    {
+        return [
+            new AuditLogTagFilterBufferedStreamHandlerPlugin(),
+        ];
+    }
+
+    /**
+     * @return list<\Spryker\Shared\Log\Dependency\Plugin\LogProcessorPluginInterface>
+     */
+    public function getProcessors(): array
+    {
+        return [
+           new PsrLogMessageProcessorPlugin(),
+           new EnvironmentProcessorPlugin(),
+           new ServerProcessorPlugin(),
+           new AuditLogRequestProcessorPlugin(),
+           new ResponseProcessorPlugin(),
+           new AuditLogMetaDataProcessorPlugin(),
+       ];
+    }
+}
+```
+
+</details>
+
+
+2. Register the plugin in the config:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Pyz\Yves\Log\Plugin\Log\YvesCheckoutAuditLoggerConfigPlugin;
+
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_YVES] = [
+    //other plugins
+    YvesCheckoutAuditLoggerConfigPlugin::class,
+];
+```
+
+3. To add the `Checkout` type for other applications, introduce `AuditLoggerConfig` plugins following the example and register them in the config file:
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Pyz\Yves\Log\Plugin\Log\YvesCheckoutAuditLoggerConfigPlugin;
+use Pyz\Glue\Log\Plugin\Log\GlueBackendCheckoutAuditLoggerConfigPlugin;
+use Pyz\Glue\Log\Plugin\Log\GlueCheckoutAuditLoggerConfigPlugin;
+use Pyz\Zed\Log\Communication\Plugin\Log\MerchantPortalCheckoutAuditLoggerConfigPlugin;
+use Pyz\Zed\Log\Communication\Plugin\Log\ZedCheckoutAuditLoggerConfigPlugin;
+
+$config[LogConstants::AUDIT_LOGGER_CONFIG_PLUGINS_ZED] = [
+    //other plugins
```

Comment:
does it mean that other plugins could be added only before the Core one?
Please mention this somewhere